hw13-dependency-injection/src/main/java/ru/otus/controllers/AdminController.java

```diff
@@ -0,0 +1,40 @@
+package ru.otus.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import ru.otus.api.model.User;
+import ru.otus.api.service.DBServiceEntityImplCached;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class AdminController {
+
+	private DBServiceEntityImplCached<User> serviceUser;
```

Comment:
интерфейс, а не реализация
и final
```suggestion
	private final DBServiceEntity<User> serviceUser;
```